responsive-web-design-piano/index12.html

```diff
@@ -6,35 +6,34 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 <body>
-    
-    <div id="piano">
-        <div class="keys">
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
 
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-        </div>
+<div id="piano">
+    <div class="keys">
+        <div class="key"></div>
     </div>
+</div>
 
 </body>
 </html>
 
+<!-- 
+    The piano will have 3 sets of 7 white keys and 5 black keys
+    on large displays. You already have one key in this set, so
+    create six more `key` `<div>` elements just below it. 
+
+    Here is an example:
+
+    ```
```

Comment:
Just realized that there are 7 strings here. Perhaps we can change this slightly so it's more like a common 6 string guitar:

```
    <div id="guitar">
        <div class="strings">
            <div class="string"></div>
            <div class="string"></div>
            <div class="string"></div>
            <div class="string"></div>
            <div class="string"></div>
            <div class="string"></div>
        </div>
    </div>
    ```